src/akkudoktoreos/devices/battery.py

```diff
@@ -10,345 +10,279 @@
 logger = get_logger(__name__)
 
 
-def max_ladeleistung_w_field(default: Optional[float] = None) -> Optional[float]:
+def max_charging_power_field(description: Optional[str] = None) -> Field:
```

Comment:
With more strict mypy this gives following error: ` Function "pydantic.fields.Field" is not valid as a type  [valid-type]` so should keep it the expected output type of the field

Reply:
ill set it to float. removing the optional since we have a default value